src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php

```diff
@@ -20,11 +20,26 @@
  */
 class SymfonyTestsListener extends \PHPUnit_Framework_BaseTestListener
 {
+    private static $globallyEnabled = false;
     private $state = -1;
     private $skippedFile = false;
     private $wasSkipped = array();
     private $isSkipped = array();
 
+    public function __construct(array $extraClockMockedNamespaces = array())
+    {
+        if ($extraClockMockedNamespaces) {
+            foreach ($extraClockMockedNamespaces as $ns) {
+                ClockMock::register($ns.'\DummyClass');
+            }
+        }
+        if (self::$globallyEnabled) {
+            $this->state = -2;
```

Comment:
but this private internal details, and constants are public... and giving a name is hard :)